doc/method/results.md

```diff
@@ -44,27 +49,26 @@ This reflects a preference among our curators for molecular
 phylogenetic evidence over other kinds.
 
 <!--
-[JAR: measure of how many mapped OTUs come from NCBI, i.e. how close NCBI 
-gets us to the mapping requirement: `../../bin/jython measure_coverage.py` = 
+[JAR: measure of how many mapped OTUs come from NCBI, i.e. how close NCBI
+gets us to the mapping requirement: `../../bin/jython measure_coverage.py` =
 NCBI 190084, OTT 195355 = 0.9730
 -->
 
 ### Taxonomic coverage
 
 OTT has 2.3M binomials (presumptive valid species names), vs. 1.6M for
-Catalogue of Life (CoL).  The number is larger in part because the
+Catalogue of Life (CoL).  Since the GBIF source we used includes the Catalogue of Life [ref], OTT includes everything in CoL.The number is larger in part because the
```

Comment:
maybe this should be 'all species' instead of 'everything' since OTT does not use CoL's upper taxonomy